coda/coda_mdstore/views.py

```diff
@@ -483,29 +484,62 @@ def bagHTML(request, identifier):
     bag = get_object_or_404(Bag, name__exact=identifier)
     bag_info = Bag_Info.objects.filter(bag_name__exact=bag)
     # determine some values that will be databased better in the future
-    payload_oxum_file_count = (
-        item for item in bag_info if item.field_name == "Payload-Oxum"
-    ).next().field_body.split(".")[1]
-    payload_oxum_size = (
-        item for item in bag_info if item.field_name == "Payload-Oxum"
-    ).next().field_body.split(".")[0]
-    bag_date = datetime.strptime((
-        item for item in bag_info if item.field_name == "Bagging-Date"
-    ).next().field_body, '%Y-%m-%d')
     try:
-        # grab the related premis events from the json search for
-        # linked object id
-        event_json_url = 'http://%s/event/search.json?linked_object_id=%s' % (
-            request.META.get('HTTP_HOST'), bag
+        payload_oxum_file_count = (
+            item for item in bag_info if item.field_name == "Payload-Oxum"
+        ).next().field_body.split(".")[1]
+    except:
+        payload_oxum_file_count = -1
+    try:
+        payload_oxum_size = (
+            item for item in bag_info if item.field_name == "Payload-Oxum"
+        ).next().field_body.split(".")[0]
+    except:
+        payload_oxum_size = -1
```

Comment:
It would be better to only iterate once. I just went with what was there. But I'll tighten it up.

I set them as negative integers to distinguish them, but I don't suppose it matters much since I didn't update the templates accordingly.